packages/design-system/src/components/Badge/style.scss

```diff
@@ -0,0 +1,28 @@
+@import '../../scss/variables/colors';
+
+.badge {
+  display: inline-block;
+  box-sizing: border-box;
+  padding: 0 3px;
+  height: 18px;
+  min-width: 18px;
+
+  font-size: 10px;
+  font-weight: 600;
+  line-height: 18px;
+  text-align: center;
+  border-radius: 18px;
+  overflow: hidden;
```

Comment:
why do we need `overflow: hidden`? when used with text it's not centered and this overflow change position of the badge https://take.ms/peZyq

Reply:
It was already there when I started work on this PR - I can remove it.